web/src/lib/components/album-page/album-viewer.svelte

```diff
@@ -86,7 +86,7 @@
 	afterNavigate(({ from }) => {
 		backUrl = from?.url.pathname ?? '/albums';
 
-		if (from?.url.pathname === '/sharing') {
+		if (from?.url.pathname === '/sharing' && album.sharedUsers.length === 0) {
```

Comment:
What does this do?

Reply:
Fix the issue with accessing an album from the sharing page doesn't render the action button on the nav bar

Reply:
Is that issue unrelated to the memory feature? If so, should it be in a separate PR?

Reply:
I could but I got annoyed by it so I fixed it here. There is no other work related to this page though so it is ok.